### PR TITLE
Increased chat icon tapping area

### PR DESCRIPTION
**Pull request by jeluard**
### Summary:

Increased the size of buton used to send messages

### Steps to test:
- Open Status
- Navigate chat
- Make sure icon to send message has a greater tapping area

status: ready

### CODE REVIEW

File: src/status_im/ui/screens/profile/events.cljs [hunk 1]
```diff
@@ -42,6 +42,7 @@
 (handlers/register-handler-fx
   :profile/send-message
   (fn [_ [_ identity]]
+    (println "IDENTITY" identity)
```
Comment (flexsurfer) [inline src/status_im/ui/screens/profile/events.cljs]: ?

File: src/status_im/ui/screens/profile/events.cljs [hunk 2]
```diff
@@ -51,7 +52,7 @@
 (handlers/register-handler-fx
   :my-profile.drawer/edit-name
   (fn [{:keys [db]} _]
-    (let [{:my-profile/keys [default-name edit auto-save]} db
+    (let [{:my-profile/keys [default-name]} db
```
Comment (flexsurfer) [inline src/status_im/ui/screens/profile/events.cljs:54]: how these changes related to this PR?

Comment (jeluard) [reply src/status_im/ui/screens/profile/events.cljs:54]: Cleaned up unused vars

**Review by goranjovic — APPROVED** [review]

File: src/status_im/ui/screens/profile/events.cljs (hunk 1 above)
Comment (jeluard) [reply src/status_im/ui/screens/profile/events.cljs]: Oups ..

**Comment by jeluard** [comment]
@flexsurfer Fixed `println` leftover

**Review by flexsurfer — APPROVED** [review]

**Comment by asemiankevich** [comment]
I can't start chat @jeluard 

image

**Comment by jeluard** [comment]
@asemiankevich Are you sure you are using latest build? This has been fixed in a recent commit.

**Comment by asemiankevich** [comment]
@jeluard triggered fresh build. the recent one i found in jenkins is 4 hours old ~.

**Pull request merged**